nautobot/dcim/filters.py

```diff
@@ -327,7 +327,7 @@ def search(self, queryset, name, value):
         )
 
 
-class RackReservationFilterSet(BaseFilterSet, TenancyFilterSet):
+class RackReservationFilterSet(BaseFilterSet, CustomFieldModelFilterSet, TenancyFilterSet):
```

Comment:
Looking at the `RackReservation` model, we should also have `CreatedUpdatedFilterSet` here - would you mind adding it?